### PR TITLE
ODH Distributed Workflow: initial e2e test setup

### PR DESCRIPTION
**Pull request by sutaakar**

### CODE REVIEW

File: ci-operator/config/opendatahub-io/distributed-workloads/opendatahub-io-distributed-workloads-main.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        sleep 10
+        make deploy-codeflare
+        sleep 120
+        cd tests && GOFLAGS=-mod=mod go test -v ./integration -run TestRayCluster
+      from: src
+      resources:
+        requests:
+          cpu: 2000m
+          memory: 3Gi
+    workflow: ipi-aws
```
Comment (anishasthana) [inline ci-operator/config/opendatahub-io/distributed-workloads/opendatahub-io-distributed-workloads-main.yaml:38]: Can we switch this over to using hypershift? https://github.com/openshift/release/pull/39476/files might be able to provide some guidance.

**Review by anishasthana — CHANGES_REQUESTED** [review]

File: ci-operator/config/opendatahub-io/distributed-workloads/opendatahub-io-distributed-workloads-main.yaml (hunk 1 above)
Comment (sutaakar) [reply ci-operator/config/opendatahub-io/distributed-workloads/opendatahub-io-distributed-workloads-main.yaml:38]: I can try it
Anyway this PR is still WIP, problem is that when I mark it as WIP then CI won't run rehearse

**Comment by sutaakar** [comment]
This PR should be ready for review now, there are some test adjustments in progress, though this PR should be already complete.

**Pull request merged**